workspace/utilities/lib/picture.xsl

```diff
@@ -20,7 +20,7 @@
 		<xsl:param name="image" select="image" />
 		<xsl:param name="alt" select="alt" />
 		<xsl:param name="sizes" select="exsl:node-set($default-image-sizes)/sizes" />
-		<xsl:param name="use-sizes" select="$image/@type != 'image/gif' and $image/@type != 'image/svg+xml'"/>
+		<xsl:param name="use-sizes" select="$image/@type != 'image/gif' and $image/@type != 'image/svg+xml' and $image/@type != 'image/tiff'"/>
```

Comment:
svg can also have the `text/plain` mime and `image/svg` . We need to add those

Reply:
https://github.com/DeuxHuitHuit/symphonycms-template/pull/365/commits/8e29837d5d7d684cd9207ea823db2e27f80ac808